accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/models/AccessionWrapper.java

```diff
@@ -21,9 +21,9 @@
  * Wrapper containing the object that has been accessioned, as well as additional information like the accession or a
  * flag indicating whether the accession is active. To be used in the service layer.
```

Comment:
There is no "flag"

Reply:
Correct, even i had commented the same https://github.com/EBIvariation/accession-commons/pull/55#issuecomment-467432597